bot/__main__.py

```diff
@@ -34,5 +34,6 @@
 bot.load_extension("bot.cogs.fun")
 bot.load_extension("bot.cogs.eval")
 bot.load_extension("bot.cogs.verification")
+bot.load_extension("bot.cogs.math")
```

Comment:
Could this be on the previous line? Not a must-have, but these lines would certainly look nicer with ascending lengths :D